src/javascript/app/common/currency.js

```diff
@@ -16,6 +16,17 @@ const getCurrencyList = (currencies) => {
     return $currencies.append($fiat_currencies.children().length ? $fiat_currencies : '').append($cryptocurrencies.children().length ? $cryptocurrencies : '');
 };
 
+const getCurrencyNameList = (currencies) => {
+    const currencies_name_list = [];
+    currencies.forEach((currency) => {
+        const is_crypto_currency = CurrencyBase.isCryptocurrency(currency);
+        const currency_name      = is_crypto_currency ? `${CurrencyBase.getCurrencyName(currency)} (${currency})` : currency;
+        currencies_name_list.push(currency_name);
```

Comment:
can we combine this somehow with `getCurrencyList`? seems they have some parts in common